lapack/gonum/idamax.go

```diff
@@ -0,0 +1,39 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gonum
+
+import (
+	"math"
+)
+
+// Idamax finds the index of the first element having maximum absolute value.
+//
+//  n is number of elements in input vector(s)
+//  dx is array, dimension ( 1 + ( N - 1 )*abs( incx ) )
+//  incx is storage spacing between elements of dx
+func (Implementation) Idamax(n int, dx []float64, incx int) (idamax int) {
```

Comment:
`Idamax` is [implemented in BLAS level 1](https://pkg.go.dev/gonum.org/v1/gonum/blas#Float64Level1), so this file is not needed.